lib/faraday/adapter/em_synchrony.rb

```diff
@@ -126,6 +117,23 @@ def execute_single_request(env, request, http_method)
           end
         end
       end
+
+      def call_block(block)
+        client = nil
+
+        if !EM.reactor_running?
```

Comment:
This would read easier if conditional was flipped to be positive.

Reply:
Good spot, I'm surprised RuboCop didn't catch that